scripts/sanitizer-env.sh

```diff
@@ -23,7 +26,11 @@ if test -n "$OUR_PRELOAD"; then
 
     LD_PRELOAD=${LD_PRELOAD+${LD_PRELOAD}:}$OUR_PRELOAD
     export LD_PRELOAD
-    echo "LD_PRELOAD has been set to \"$LD_PRELOAD\"."
-    echo "The sanitizer might report issues with ANY process you execute."
+    cat <<- EOF
+
+    LD_PRELOAD has been set to \"$LD_PRELOAD\"."
```

Comment:
One leftover `"`.... But you also don't need to escape them, so just remove the escapes and the extra one

Reply:
Nicely spotted. Updated + added some quoted the `OUR_PRELOAD` message when it's not a file.